packages/iov-faucets/README.md

```diff
@@ -0,0 +1,14 @@
+# @iov/faucets
+
+[![npm version](https://img.shields.io/npm/v/@iov/faucets.svg)](https://www.npmjs.com/package/@iov/faucets)
+
+Use some Testnet faucets from JavaScript.
+
+## API Documentation
```

Comment:
Maybe you can add a two liner example here as well, not just in cli.

```
const bovFaucet = new BovFaucet("https://faucet.friendnet-slow.iov.one/faucet");
await bovFaucet.open(addr)
```

Sure, you need to set an address and query and all that, but this shows main usage quite simply.

Reply:
done